### PR TITLE
create login page

### PR DESCRIPTION
**Pull request by Bluewizzy1**
create login page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,17 +3,19 @@ import { Routes, Route } from "react-router-dom";
...
 
 import SignUp from "./pages/SignUp";
+import Login from "./pages/Login";
 import Dashboard from "./pages/Dashboard";
 import Topbar from "./components/Topbar";
 
 function App() {
   return (
-    <div className="bg-[#D2D2D2]">
+    <div className="bg-[#D2D2D2] w-full max-w-[fixed-width] px-[30px]">
```
Comment (tiammui) [inline src/App.js]: The **_fixed-width_** here should have being the container width in pixels e.g 100px, 20px etc

File: src/pages/Login.js
Comment (tiammui) [inline src/pages/Login.js:1]: We should go with this for now, but this file will be worked on later
It looks good, but we'll face problem when we want to integrate functionalities later

**Review by tiammui — CHANGES_REQUESTED** [review]

**Comment by Bluewizzy1** [comment]
What is the cause for the potential challenges

**Comment by tiammui** [comment]
We can discuss the potential issue on a call 🙂

I requested some changes on some part of the code

Please action it so that this PR can be merged

**Pull request merged**